genfragments/ThirteenTeV/Hadronizer/Hadronizer_TuneCUETP8M1_13TeV_EMEnrichedFilter_LHE_pythia8_cff.py

```diff
@@ -0,0 +1,70 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
+
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
+                                     args=cms.vstring(
+                                         'GRIDPACK'),
+                                     nEvents=cms.untracked.uint32(5000),
+                                     numberOfParameters=cms.uint32(1),
+                                     outputFile=cms.string(
+                                         'cmsgrid_final.lhe'),
+                                     scriptName=cms.FileInPath(
+                                         'GeneratorInterface/LHEInterface/data/run_generic_tarball_cvmfs.sh')
+                                     )
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+                         maxEventsToPrint=cms.untracked.int32(1),
+                         pythiaPylistVerbosity=cms.untracked.int32(1),
+                         filterEfficiency=cms.untracked.double(1.0),
+                         pythiaHepMCVerbosity=cms.untracked.bool(False),
+                         comEnergy=cms.double(13000.),
+                         PythiaParameters=cms.PSet(
+                             pythia8CommonSettingsBlock,
+                             pythia8CUEP8M1SettingsBlock,
+                             processParameters=cms.vstring(
+                                 'JetMatching:setMad = off',
+                                 'JetMatching:scheme = 1',
+                                 'JetMatching:merge = on',
+                                 'JetMatching:jetAlgorithm = 2',
+                                 'JetMatching:etaJetMax = 5.',
+                                 'JetMatching:coneRadius = 1.',
+                                 'JetMatching:slowJetPower = 1',
+                                 'JetMatching:qCut = 30.',
+                                 'JetMatching:nQmatch = 5',
+                                 'JetMatching:nJetMax = 1',
+                                 'JetMatching:doShowerKt = off',
```

Comment:
Why do you use matching? Matching assumes that you get +1 jet contributions from 0 jets @ ME + PS. However, this is not produced.